Ports/mc/package.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=mc
+useconfigure=true
+version=4.8.27
+configopts=("--without-x" "--disable-largefile" "--with-homedir" "--disable-vfs" "-with-screen=ncurses" "--without-edit" "--with-ncurses-includes=$SERENITY_BUILD_DIR/Root/usr/local/include/ncurses" "--with-ncurses-libs=$SERENITY_BUILD_DIR/Root/usr/local/lib")
+depends=("gettext" "libtool" "ncurses" "glib" "vim")
+files="https://github.com/MidnightCommander/mc/archive/refs/tags/${version}.tar.gz ${port}-${version}.tar.gz 3bab1460d187e1f09409be4bb8550ea7dab125fb9b50036a8dbd2b16e8b1985b"
+auth_type=sha256
```

Comment:
Let's shuffle these around a bit, and make `configopts` much more readable:

```suggestion
port=mc
version=4.8.27
useconfigure=true
files="https://github.com/MidnightCommander/mc/archive/refs/tags/${version}.tar.gz ${port}-${version}.tar.gz 3bab1460d187e1f09409be4bb8550ea7dab125fb9b50036a8dbd2b16e8b1985b"
auth_type=sha256
depends=("gettext" "libtool" "ncurses" "glib" "vim")
configopts=(
    "--disable-largefile"
    "--disable-vfs"
    "--without-edit"
    "--without-x"
    "--with-homedir"
    "-with-screen=ncurses"
    "--with-ncurses-includes=$SERENITY_BUILD_DIR/Root/usr/local/include/ncurses"
    "--with-ncurses-libs=$SERENITY_BUILD_DIR/Root/usr/local/lib"
)
```

Also, is `"-with-screen=ncurses"` a typo?

Reply:
That sure was typo. Re-ordered as suggested